src/Constants.js

```diff
@@ -0,0 +1,12 @@
+export const PAYMENT_SUBMIT_EVENT = 'didSubmitCallback';
+export const PAYMENT_PROVIDE_DETAILS_EVENT = 'didProvideCallback';
+export const PAYMENT_COMPLETED_EVENT = 'didCompleteCallback';
+export const PAYMENT_FAILED_EVENT = 'didFailCallback';
+
+export const ERROR_CODE_CANCELED = 'canceledByShopper';
+export const ERROR_CODE_NOT_SUPPORTED = 'notSupported';
+export const ERROR_CODE_NO_CLIENT_KEY = 'noClientKey';
+export const ERROR_CODE_NO_PAYMENT = 'noPayment';
+export const ERROR_CODE_INVALID_PAYMENT_METHODS = 'invalidPaymentMethods';
+export const ERROR_CODE_INVALID_ACTION = 'invalidAction';
+export const ERROR_CODE_NO_PAYMENT_METHOD = 'noPaymentMethod';
```

Comment:
If the file isn't a component, then better to name it lowercase to `constants.js` . Usually naming PascalCase means that the file is a UI Component

Reply:
Fixed in [8c66d33](https://github.com/Adyen/adyen-react-native/pull/52/commits/8c66d33390ca8435f367270e62d1e9ee7e6b2cd5)